third_party/move/move-compiler-v2/tests/bytecode-generator/wildcard2.exp

```diff
@@ -0,0 +1,30 @@
+// -- Model dump before bytecode pipeline
+module 0xc0ffee::m {
+    public fun baz() {
+        {
+          let x: u64;
+          {
+            let _: u64 = x;
+            Tuple()
+          }
+        }
+    }
+} // end 0xc0ffee::m
+
+============ initial bytecode ================
+
+[variant baseline]
+public fun m::baz() {
+     var $t0: u64
+     var $t1: u64
+  0: $t1 := infer($t0)
+  1: return ()
+}
+
+
+Diagnostics:
+error: use of unassigned local `x`
+  ┌─ tests/bytecode-generator/wildcard2.move:4:13
+  │
+4 │         let _ = x;
+  │             ^
```

Comment:
Why does the error point to `_` instead of `x`?

Reply:
Filed an issue for this: https://github.com/aptos-labs/aptos-core/issues/12843